initproject.js

```diff
@@ -0,0 +1,17 @@
+import cp from 'child_process';
+import path from 'path';
+
+const APP_NAME_TOKEN = '{{APP_NAME}}';
+const APP_NAME = path.basename(process.cwd());
+
+// replace occurrences of app name token
+console.log(`Setting app name as "${APP_NAME}"...`);
+cp.execSync(`LC_ALL=C find . -type f ! -name "initproject.js" -exec sed -i '' s/${APP_NAME_TOKEN}/${APP_NAME}/g {} +`);
+// TODO use template to replace README.md contents w/app-specific info
+// install deps
+console.log(`Installing dependencies, this may take awhile...`);
+cp.execSync('yarn install');
+// initialize git repo
+console.log(`Initializing new git repo...`);
+cp.execSync('git init');
+console.log("🍻 Your brand spankin' new React app is ready!");
```

Comment:
the node script that sets up the project after being cloned from the blueprint. Would like to eventually make this more interactive/robust, and maybe its own little real cli that we can invoke with `npx` or w/e, thats more in the vein of ember-cli's `ember new ...` command.